### ISSUE TITLE
Clicking on breadcrumb of active location causes vue error

### ISSUE THREAD

**Issue by michalsieron** [agersant/polaris-web#63]
As in title. I didn't notice any visible effects of this error, but thought you should know about this and maybe ignore such actions.

```js
Uncaught (in promise) ▸ Object { _name: "NavigationDuplicated", name: "NavigationDuplicated", message: "Navigating to current location (\"/browse/\") is not allowed", stack: "", … } vue-router.esm.js:2089:8
```

Polaris is running in docker container from [ogarcia/docker-polaris](https://github.com/ogarcia/docker-polaris).
Accessed on Firefox 79.0

**Comment by michalsieron** [comment]
Same appplies to tabs in Settings

**Comment by agersant** [comment]
Good find! 

Fixed in acce5403ed8df4f10cd503a07f06b703ac97c921
